fail2ban/tests/dummyjail.py

```diff
@@ -24,43 +24,66 @@
 
 from threading import Lock
 
+from ..server.jail import Jail
 from ..server.actions import Actions
 
 
-class DummyJail(object):
+class DummyJail(Jail, object):
```

Comment:
no need for ", object", if Jail is not a "new style" class (we should migrate all to anyways) -- make it `(object)` and be done